spl-ids/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "solana-spl-ids"
```

Comment:
nit: Since the crate has more than just ids, how about `solana-inline-spl`?

Reply:
done